tools/HelperUtilities/psScripts/Copy-SessionRecords.ps1

```diff
@@ -0,0 +1,73 @@
+# ---------------------------------------------------------------------------------- 
+    # Copyright (c) Microsoft Corporation. All rights reserved.
+    # Licensed under the MIT License. See License.txt in the project root for
+    # license information.
+# ---------------------------------------------------------------------------------- 
+
+<#
+
+.SYNOPSIS
+    Powershell script that copies newly recorded session records from source to destination
+
+.DESCRIPTION
+    This script copies newly recorded session records from source to destination
+
+.PARAMETER SourceRootDirectory
+    Full path to directory where session records are (usually the bin directory of recorded test)
+
+.PARAMETER DestinationRootDirectory
+    Location where to copy the files
+#>
+
+[cmdletbinding(SupportsShouldProcess=$True)]
+Param(
+    [Parameter(Mandatory = $true)]
+    [string] $SourceRootDirectory,
+    [Parameter(Mandatory = $true)]
+    [string] $DestinationRootDirectory
+)
+
+Begin {
+    if (-not $PSBoundParameters.ContainsKey('Confirm')) {
+        $ConfirmPreference = $PSCmdlet.SessionState.PSVariable.GetValue('ConfirmPreference')
```

Comment:
@dsgouda let's not tie the logic to execute -Confirm with ConfirmPreference PSVariable.
Similarly also let's not tie WhatIf with WhatIfPreference